modules/thermal_hydraulics/doc/content/source/materials/ADWallHeatTransferCoefficientWeismanMaterial.md

```diff
@@ -0,0 +1,35 @@
+# ADWallHeatTransferCoefficientWeismanMaterial
+
+The material computes the convective heat transfer coefficient for water in a rod bundle using the Weisman correlation.
+
+Equations (10.92b), (10.104a), and (10.104b)  from [!cite](todreas2021nuclear) are used.
+
+The Nusselt number is calculated as:
+
+\begin{equation}
+  Nu = \begin{cases}
+      0.023(Re^{0.8} Pr^n)\left[1.826 \left(\frac{P}{D}\right) - 1.0430 \right] & \text{for a square array rod bundle}\\
+      \\
+      0.023(Re^{0.8} Pr^n)\left[1.130 \left(\frac{P}{D}\right) - 0.2609 \right] & \text{for a triangular array rod bundle}\\
+    \end{cases}
+\end{equation}
+
+with the Reynolds, $Re$, and Prandtl, $Pr$, numbers given by:
+
+\begin{equation}
+  Re = \frac{\rho v D_h}{\mu},\text{ and } Pr = \frac{c_p \mu}{k}
+\end{equation}
+
+where $P/D$ is the pitch-to-diameter ratio, and $c_p$ is the heat capacity, $\rho$ is the density, $\mu$ is the viscosity, $k$ is the fluid thermal conductivity, and $D_h$ is the hydraulic diameter. The constant $n$ is equal to $0.4$ for heating problems, and $0.3$ for cooling problems.
+
+The Weisman correlation is valid for $1.1\leq P/D \leq 1.3$ for the square array, $1.1\leq P/D \leq 1.5$ for the triangular array, $0.7<Pr<100$, and $Re>10000$. The convective heat transfer coefficient, $h$, is calculated as:
```

Comment:
It's unclear what the Pr and Re ranges apply to.

Reply:
OK, cool, Wesiman's correlation is valid for 0.7<Pr<100, and $Re>10000 for both cases. I will rewrite this to make clear that the  range applies to both arrays.